modules/cluster-logging-updating-logging.adoc

```diff
@@ -163,3 +158,22 @@ $ oc get CronJob curator -n openshift-logging -o yaml |grep 'image:'
 ----
 image: registry.redhat.io/openshift4/ose-logging-curator5@sha256:330c3499e790d0e184414125a4843cd48849c601eb9f19ff82f30794c858b0bc
 ----
+
+.Post install
+
+After Cluster Logging and Elasticsearch Operators are fully updated to 4.5, you must perform the following actions to complete the upgrade.
+
```

Comment:
Lines 165 to 173 are useless.   User must  create index patterns in kibana after upgrade